public/docs/getting_started/console.md

```diff
@@ -1,36 +1,137 @@
-# Using the console
+# Using Truffle Develop and the console
 
-Sometimes it's nice to work with your contracts interactively for testing and debugging purposes, or for executing transactions by hand. Truffle provides you an easy way to do this via an interactive console, with your contracts available and ready to use.
+Sometimes it's nice to work with your contracts interactively for testing and debugging purposes, or for executing transactions by hand. Truffle provides you two easy ways to do this via an interactive console, with your contracts available and ready to use.
 
-## Command
+* **Truffle console**: A basic interactive console
+* **Truffle Develop**: An interactive console that also spawns a development blockchain
 
-To fire up the console, simply run:
+## Why two different consoles?
 
-```none
-$ truffle console
-```
+Having two different consoles allows you to choose the best tool for your needs.
+
+Reasons to use Truffle console:
+
+* You have a client you're already using, such as [Ganache](/docs/ganache/using) or geth
+* You want to migrate to a testnet (or the main Ethereum network)
+* You want to use a specific mnemonic or account list
+
+Reasons to use Truffle Develop:
+
+* You are testing your project with no intention of immediately deploying
+* You don't need to work with specific accounts
+* You don't need or want a separate blockchain running
 
-This will load up a console using the `development` network, if specified. You can override this using the `--network` option. See more details in the [Networks](/docs/advanced/networks) section as well as the [command reference](/docs/advanced/commands).
+## Commands
 
-When you load the console, you'll immediately see output like this:
+All commands require that you be in the root of your project folder.
```

Comment:
I don't think this is actually true. Truffle is smart enough to look up into your directories for a `truffle.js` file. Give it a shot and let me know what you find. e.g., if you `cd` into your `contracts` directory, then run `truffle console`, it _should_ load.

Reply:
Ah right you are. I ran in the `contracts` directory and it ran just fine. Good to know.